src/mam4xx/tropopause.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef MAM4XX_TROPOPAUSE_HPP
+#define MAM4XX_TROPOPAUSE_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/aero_config.hpp>
+
+#include <fstream>
+#include <iostream>
+
+namespace mam4 {
+
+namespace tropopause {
+
+using ConstColumnView = haero::ConstColumnView;
+// From radconstants
+constexpr int nswbands = 14;
+constexpr int nlwbands = 16;
+using View2D = DeviceType::view_2d<Real>;
+constexpr Real km_inv_to_m_inv = 0.001; // 1/km to 1/m
+
+constexpr Real shr_const_rgas =
+    haero::Constants::r_gas * 1e3; // Universal gas constant ~ J/K/kmole
```

Comment:
We've been using the notation `[J/K/kmole]` to indicate the units for a constant or quantity.